migrations/0236_bid_due_date_report.sql

```
@@ -0,0 +1,178 @@
+    'bid_due_date_report_cc',
+    'Bid Due Date Report (Cc)',
+    'Copied on the weekly Wednesday estimating report of upcoming bid due dates.'
+  )
+ON CONFLICT (key) DO UPDATE
+SET
+  name = EXCLUDED.name,
+  description = EXCLUDED.description;
+
+-- TO: the estimator the report is written for.
+INSERT INTO public.notification_recipient_assignments (group_id, user_id)
+SELECT g.id, u.id
```

SQLFluff (4.3.0)

[error] Reuse of column alias 'id' from line 123.

(AL08)